crates/configuration/src/network.rs

```diff
@@ -1,35 +1,53 @@
+use core::marker::PhantomData;
+
+use serde::Serialize;
+
 use crate::{
-    shared::types::{Duration, IpAddress, MultiAddress},
+    errors::ConfigError,
+    shared::types::{IpAddress, MultiAddress, ParaId, TimeoutInSecs},
     HrmpChannelConfig, ParachainConfig, RelaychainConfig,
 };
 
+#[derive(Default, Clone)]
+pub struct NoRelayChain;
+#[derive(Default, Clone)]
+pub struct WithRelayChain;
+
 /// Global settings applied to an entire network.
+#[derive(Debug, Serialize)]
 pub struct GlobalSettings {
+    /// [TODO]: This feature is not well defined, which binary/image should use to spawn the bootnode?
```

Comment:
I would say the default image/binary on a parachain or relaychain, it makes this property dependant on the default_image/default_command on underlying type in validation ?

Reply:
Yes, we can use that approach and add the docs to explain the expected behavior. At the moment, in the current version, we take the image from the `first` node in the relaychain. I think is ok, but we should add a nice documentation.
Thanks!

Reply:
I think it is clearer to the user (better DX) to have an extra param in the configuration called "create_bootnode"; If used then the node with that param will be used for creating the bootnodes; If not then by default the `first` node in the relaychain